stable/mongodb/README.md

```diff
@@ -43,36 +43,52 @@ The command removes all the Kubernetes components associated with the chart and
+| `persistence.size`                      | Size of data volume                                                                          | `8Gi`                                                    |
+| `persistence.annotations`               | Persistent Volume annotations                                                                | `{}`                                                     |
+| `livenessProbe.initialDelaySeconds`     | Delay before liveness probe is initiated                                                     | `30`                                                     |
+| `livenessProbe.periodSeconds`           | How often to perform the probe                                                               | `10`                                                     |
+| `livenessProbe.timeoutSeconds`          | When the probe times out                                                                     | `5`                                                      |
+| `livenessProbe.successThreshold`        | Minimum consecutive successes for the probe to be considered successful after having failed. | `1`                                                      |
+| `livenessProbe.failureThreshold`        | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | `6`                                                      |
+| `readinessProbe.initialDelaySeconds`    | Delay before readiness probe is initiated                                                    | `5`                                                      |
+| `readinessProbe.periodSeconds`          | How often to perform the probe                                                               | `10`                                                     |
+| `readinessProbe.timeoutSeconds`         | When the probe times out                                                                     | `5`                                                      |
+| `readinessProbe.failureThreshold`       | Minimum consecutive failures for the probe to be considered failed after having succeeded.   | `6`                                                      |
+| `readinessProbe.successThreshold`       | Minimum consecutive successes for the probe to be considered successful after having failed. | `1`                                                      |
+| `configmap`                             | MongoDB configuration file to be used                                                        | `nil`                                                    |
 
 The above parameters map to the env variables defined in [bitnami/mongodb](http://github.com/bitnami/bitnami-docker-mongodb). For more information please refer to the [bitnami/mongodb](http://github.com/bitnami/bitnami-docker-mongodb) image documentation.
```

Comment:
we should remove this line. It does not make sense anymore.
We need to add a section on deploying the chart with values-production.yaml configuration (see https://github.com/kubernetes/charts/tree/master/stable/wordpress#production-and-horizontal-scaling)

Reply:
done